CHANGELOG

```diff
@@ -12,6 +12,10 @@ v3.17 (Month 2020)
     - New form-view to edit each field individually
     - Formatting toolbar to help with markup
     - Side-by-side editor preview that auto-updates
+  * Improve current implementation of persisting data when creating and editing Card, Evidence, Issue, and Note
```

Comment:
This is a very technical description of what we're doing.

The user cares more about not loosing data than "implementation" or "persisting", etc.

Can you please reword in a way that the change can be easily understood by a non-developer user?

Reply:
Okay, will do!